api/spec/src/productcatalog/subscription.tsp

```diff
@@ -233,6 +233,18 @@ model SubscriptionEdit {
   customizations: SubscriptionEditOperation[];
 }
 
+/**
+ * Response body for subscription change.
+ */
+@friendlyName("SubscriptionChangeResponseBody")
+model SubscriptionChangeResponseBody {
+  @summary("Current subscription")
+  current: Subscription;
```

Comment:
Should we call this `previous` and `next` instead?
Current, to me, is ambiguous as we don't know when the previous ends.

Reply:
I suppose thats fair, previous is weird for me as if the change is scheduled later previous would point to a still active one that might deactivate in days or later (thats why i chose current). New vs Next should definitely be Next though (IMO)